holder.go

```diff
@@ -52,7 +52,7 @@ type Holder struct {
 
 	// Key/ID translation
 	translateFile            *TranslateFile
-	NewPrimaryTranslateStore func(interface{}) TranslateStore
+	NewPrimaryTranslateStore func(interface{}) (TranslateStore, error)
```

Comment:
We can't actually make this change since it's an exported field. Luckily, I don't believe we actually need to. See my comment in the implementation below.